packages/Heading/stories/examples/HeadingLevels.js

```diff
@@ -80,12 +80,6 @@ const HeadingLevelsStory = () => {
       <Heading level={6} isLight>
         Heading Six with isLight
       </Heading>
-      <br />
```

Comment:
This is adding propTypes validation errors. and no adding any benefit to the stories.